apps/prairielearn/src/lib/file-store.js

```diff
@@ -17,7 +18,7 @@ const StorageTypes = Object.freeze({
  * Upload a file into the file store.
  *
  * @param {string} display_filename - The display_filename of the file.
- * @param {Buffer} contents - The file contents.
+ * @param {Buffer} buffer - The file contents.
```

Comment:
Nit: I actually like `contents` better as an option.

Reply:
I have now updated the usage in the function to `contents` instead of `buffer`.